Interni/NormeDiProgetto/res/sections/processi_organizzativi/gestione_di_processo.tex

```diff
@@ -0,0 +1,207 @@
+    \item \textbf{creazione issue di esecuzione}: il responsabile crea una issue su GitHub secondo quanto riportato nel capitolo sulla "Gestione dei cambiamenti". Provvede ad inserire nella descrizione la data di scadenza precedentemente stabilità, ed eventualmente una breve descrizione del lavoro da svolgere;
+    \item \textbf{spostamento issue in "TO-DO"}: il responsabile sposta la issue nella colonna "TO-DO" all'interno della board di progetto, così da identificarla come da svolgere;
+    \item \textbf{spostamento issue in "IN PROGRESS"}: una volta che un assegnatario decide di iniziare il lavoro dopo essersi autoassegnato la issue, o dopo che il responsabile di progetto gli abbia incaricato l'attività, provvederà a spostarla nella colonna "IN PROGRESS";
+    \item \textbf{spostamento issue in "DONE"}: terminato il lavoro l'assegnatario provvederà a spostare la issue in "DONE" ed a chiudere la issue;
+    \item \textbf{creazione pull request}: l'assegnatario a questo punto dovrà creare una pull request dal ramo contenente il lavoro svolto al ramo dove dovranno essere applicate queste modifiche. Dovrà poi indicare nel campo "Reviewer" un componente incaricato della verifica ed inserire la lable "da revisionare". Può aggiungere alla pull request una descrizione contenente dei commenti sul lavoro svolto se ritiene che ciò possa essere utile al controllo da parte del verificatore. Conclude inserendo la pull request all'interno della sezione "TO-DO" della board di progetto;
+    \item \textbf{spostamento pull request in "IN PROGRESS"}: il verificatore quando inizia il lavoro di verifica sposta la pull request in "IN PROGRESS";
+    \item \textbf{creazione feedback di verifica}: terminata la verifica il verificatore scrive un feedback del lavoro sottopostogli, e richiede delle modifiche nel caso riscontri delle correzioni da fare. In questo caso assegna la pull request al componente del gruppo che la creata, e modifica la label in "da correggere";
+    \item \textbf{svolgimento modifiche}: l'assegnatario dopo aver ricevuto dal verificatore un feedback sulle cose da sistemare provvederà a svolgerle. Una volta terminate le correzioni richiederà un'ulteriore verifica attraverso il tasto "re-reviews" e modificherà la lable in "da revisionare";
+    \item \textbf{spostamento pull request in "DONE"}: se il verificatore non riscontra correzioni da fare, o quelle effettuate sono sodisfacenti, sposta la pull request in "DONE" ed elimina l'etichetta ad essa applicata;
+    \item \textbf{approvazione del lavoro}: il responsabile di progetto dopo aver esaminato il lavoro lo approva effettuando il merge con il ramo interessato. Nel fare ciò dovrà selezionare l'opzione "Squash and marge" ed assegnare un titolo significativo al commit.
```

Comment:
lo squash dei commit avviene quando un documento passa la verifica e converge in develop.
Non è necessario l'intervento del responsabile di progetto, se i verificatori decidono che il documento è a posto chiudono la pulll request e mandano tutto in develop (almeno mi sembra di aver capito che facciamo cosi)

Reply:
Sì giusto, dopo sistemo. 
Quindi tolgo proprio del tutto il punto **Approvazione del lavoro**, o lo lascio e scrivo soltanto: _"il responsabile di progetto dopo aver esaminato il lavoro lo approva"_ cioè fa scattare la versione per il rilascio per intendersi, oppure lascio stare e non serve scriverlo qua?